pom.xml

```diff
@@ -79,6 +79,11 @@
       <artifactId>ical4j</artifactId>
       <version>3.0.1</version>
     </dependency>
+	<dependency>
```

Comment:
you can use `xmllint --format pom.xml --output pom.xml` to reformat the file :)

Reply:
Ok.